.github/workflows/changeset-release.yml

```diff
@@ -49,3 +66,10 @@ jobs:
           GIT_AUTHOR_EMAIL: github-actions[bot]@users.noreply.github.com
           GIT_COMMITTER_NAME: github-actions[bot]
           GIT_COMMITTER_EMAIL: github-actions[bot]@users.noreply.github.com
+
+      - name: Publish to npm (if not already published)
```

Comment:
How is the "if not already published" checked here? I guess it's already handled by [`changeset publish`](https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#publish)?

Reply:
Correct. It checks for `npm info` and bails if the version already exists on npm

See logs here: https://github.com/iTwin/iTwinUI/actions/runs/8249790380/job/22562943074